hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/replication/health/TestEmptyContainerHandler.java

```diff
@@ -63,9 +63,18 @@ public void setup()
         HddsProtos.ReplicationFactor.THREE);
     replicationManager = Mockito.mock(ReplicationManager.class);
     containerManager = Mockito.mock(ContainerManager.class);
+    
+    Mockito.doAnswer(invocation -> {
```

Comment:
Do we need this part? If we remove the ContainerManager dependency, then we don't really need to have it in this test class anymore and we can just verify the ReplicationManager.updateContainerState is called and leave it there.

Reply:
thanks, removed containerManager from the test